Cargo.toml

```diff
@@ -42,12 +42,13 @@ gethostname = "0.2.1"
 chrono = { version = "0.4.19", features = ["serde"] }
 orion = "0.16.0"
 hex = "0.4.3"
-# tako = { path = "../tako" }
```

Comment:
Temporarily it would be better to select specific SHA in the remote repository, otherwise CI won't be able to build the PR.

Reply:
Ah good point